devimint/src/main.rs

```diff
@@ -152,7 +152,7 @@ async fn cli_tests(dev_fed: DevFed) -> Result<()> {
         "--in-file={data_dir}/server-0/private.encrypt",
         "--out-file={data_dir}/server-0/config-plaintext.json"
     )
-    .env("FM_PASSWORD", "pass0")
+    .env("FM_PASSWORD", "pass")
```

Comment:
Note: we now are using the same password for every server in the devimint environment